CourseGrab/ViewControllers/Search/SearchTableViewCell.swift

```diff
@@ -12,6 +12,7 @@ class SearchTableViewCell: UITableViewCell {
 
     private let arrowImageView = UIImageView()
     private let containerView = UIView()
+    private let headerStackView = UIStackView()
```

Comment:
Is there still a need for the `UIStackView`? Since there is only one label in it right now.

Reply:
We don't but it is a cleaner solution for if we wanted to add the professor label or any other label in the future and still have arrow centered between all the labels.